apps/guide/settings/base.py

```diff
@@ -190,12 +190,60 @@
 USE_TZ = True
 
 WAGTAIL_CONTENT_LANGUAGES = LANGUAGES = [
-    ("en-latest", "English latest"),
-    ("en-3.1.x", "English 3.0.x"),
-    ("nl-latest", "Dutch latest"),
-    ("nl-3.1.x", "Dutch 3.0.x"),
-    ("hi-latest", "Hindi latest"),
-    ("hi-3.1.x", "Hindi 3.0.x"),
+    ("af", "Afrikaans"),
+    ("ar", "Arabic"),
+    ("az-AZ", "Azerbaijani (Azerbaijan)"),
+    ("be", "Belarusian"),
+    ("bg", "Bulgarian"),
+    ("bn", "Bengali"),
+    ("ca", "Catalan"),
+    ("cs", "Czech"),
+    ("cy", "Welsh"),
+    ("da", "Danish"),
+    ("de", "German"),
+    ("el", "Greek"),
+    ("en", "English"),
+    ("es", "Spanish"),
+    ("et", "Estonian"),
+    ("eu", "Basque"),
+    ("fa", "Persian"),
+    ("fi", "Finnish"),
+    ("fr", "French"),
+    ("gl", "Galician"),
+    ("he-IL", "Hebrew (Israel)"),
+    ("hr-HR", "Croatian (Croatia)"),
+    ("ht", "Haitian (Haitian Creole)"),
+    ("hu", "Hungarian"),
+    ("hy", "Armenian"),
+    ("id-ID", "Indonesian (Indonesia)"),
+    ("is-IS", "Icelandic (Iceland)"),
+    ("it", "Italian"),
+    ("ja", "Japanese"),
+    ("ka", "Georgian"),
+    ("lt", "Lithuanian"),
+    ("lv", "Latvian"),
+    ("mi", "Maori"),
+    ("mn", "Mongolian"),
+    ("my", "Burmese"),
+    ("nb", "Norwegian Bokmål"),
+    ("pl", "Polish"),
+    ("pt-BR", "Portuguese (Brazil)"),
+    ("ro", "Romanian"),
+    ("ru", "Russian"),
+    ("sk-SK", "Slovak (Slovakia)"),
+    ("sl", "Slovenian"),
+    ("sr", "Serbian"),
+    ("sv", "Swedish"),
+    ("ta", "Tamil"),
+    ("tet", "Tetum (Tetun)"),
+    ("th", "Thai"),
+    ("tr", "Turkish"),
+    ("tr-TR", "Turkish (Turkey)"),
+    ("uk", "Ukrainian"),
+    ("vi", "Vietnamese"),
+    ("zh", "Chinese"),
+    ("zh-Hant", "Chinese Traditional"),
+    ("zh-CN", "Chinese (China)"),
```

Comment:
Same as my other remark. We need the `-latest` and `-X.X.X` if we want to present versioned content.